js/PageLevelProgressNavigationView.js

```diff
@@ -13,6 +13,12 @@ export default class PageLevelProgressNavigationView extends Backbone.View {
     return 'btn-icon nav__btn nav__pagelevelprogress-btn pagelevelprogress__nav-btn';
   }
 
+  attributes() {
+    return {
+      'data-order': (Adapt.course.get('_globals')?._extensions?._pageLevelProgress?._navOrder || 90)
```

Comment:
Code is defaulting to 90, so should the schemas also use 90 for the default?

Reply:
Ach no, I'll set to 0. because float:right inverts the current visual order, reordering these correctly in the dom will currently produce the wrong visual order. These should default to 0 but at least be available to change when we move to floats.